src/modules/simulator/simulator.h

```diff
@@ -276,7 +276,7 @@ class Simulator : public ModuleParams
 
 	int publish_sensor_topics(mavlink_hil_sensor_t *imu);
 	int publish_flow_topic(mavlink_hil_optical_flow_t *flow);
```

Comment:
Can you use a `const` reference for the reset as well?

Reply:
Done! Thanks @bkueng ! (I almost did that yesterday, but I figured doing less at first was the better option to start with. I was happy to see your comment!)